Source/WebCore/xml/XPathUtil.cpp

```diff
@@ -73,5 +74,10 @@ bool isValidContextNode(Node& node)
     return false;
 }
 
+bool isXMLSpace(UChar character)
```

Comment:
XPath code seems like the wrong place fro this generic function. I know that Anne and Darin have been discussing related cleanup recently, not sure how closely that's related.

Reply:
@annevk do you have a suggestion for a better location for this function?

Reply:
`ASCIICType.h` is a good location I think, following the recent move of `isHTMLSpace` (plus renaming) there, but then the name should be either `isHTTPWhitespace` or `isASCIIWhitespaceMinusFormFeed`/`isASCIIWhitespaceMinusFF`.  We'll primarily use it for HTTP, MIME types, and such, but apparently also for some XML-related features that are not covered by libxml.

Reply:
Will wait for @annevk's response and will work accordingly.

Reply:
I think `isASCIIWhitespaceMinusFormFeed` is clearer name to me.

Reply:
@aproskuryakov & @cdumez - I am able to do this in local build but my PR Git setup is bad and if I rebased then it will mess-up. I am happy to do follow-up to fix address this. My local branch is outdated and don't have @annevk's PR, so it will lead to merge conflict, if I do and rebasing tends to also mess-up this commit. Or I can close this PR and do new with all changes in one go.

Reply:
Only issue is that @annevk has few open PRs, so I can do once all of them are merged and remove 'XMLSpace' from these functions and add them to 'ASCIICType.h' as suggested later.